README.md

```diff
@@ -90,6 +91,78 @@ Sure! However you will need a [SPID account](https://www.agid.gov.it/en/platform
+| NSAppleMusicUsageDescription                | This key is required if your app uses APIs that access the user’s media library.                                                         |
+| NSBluetoothAlwaysUsageDescription           | This key is required if your app uses the device’s Bluetooth interface.                                                                  |
+| NSBluetoothPeripheralUsageDescription       | This key is required if your app uses APIs that access Bluetooth peripherals and has a deployment target earlier than iOS 13.            |
+| NSContactsUsageDescription                  | IO needs access to your contacts to let you add them in calendar events.                                                                 |
+| NSLocationAlwaysUsageDescription            | This key is required if your iOS app uses APIs that access the user’s location at all times and deploys to targets earlier than iOS 11.  |
+| NSLocationUsageDescription                  | This key is required if your app uses APIs that access the user’s location information.                                                  |
+| NSLocationWhenInUseUsageDescription         | This key is required if your iOS app uses APIs that access the user’s location information while the app is in use.                      |
+| NSMicrophoneUsageDescription                | IO needs access to the microphone in case you want to leave a voice note.                                                                |
+| NSMotionUsageDescription                    | This key is required if your app uses APIs that access the device’s motion data.                                                         |
+| NSCalendarsUsageDescription                 | IO needs access to the calendar to add event reminders.                                                                                  |
+| NSCameraUsageDescription                    | IO needs access to the camera to scan QR codes.                                                                                          |
+| NSFaceIDUsageDescription                    | Enable Face ID for a faster identification.                                                                                              |
+| NSPhotoLibraryAddUsageDescription           | This key is required if your app uses APIs that have write access to the user’s photo library.                                           |
+| NSPhotoLibraryUsageDescription              | IO needs access to the Photos to scan QR codes.                                                                                          |
+| NSSpeechRecognitionUsageDescription         | IO needs access to the Photos to scan QR codes.                                                                                          |
```

Comment:
[NSSpeechRecognitionUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nsspeechrecognitionusagedescription) should have a different usage description:

> This key is required if your app uses APIs that send user data to Apple’s speech recognition servers.

Reply:
```suggestion
| NSSpeechRecognitionUsageDescription         | This key is required if your app uses APIs that send user data to Apple’s speech recognition servers. Used in the assistance flow.                                                                                        |
```